### PR TITLE
Removed unused constraints

### PR DESCRIPTION
**Pull request by Sathawale27**
Removed unused constraints.

### CODE REVIEW

File: app/src/main/res/layout/activity_home.xml [hunk 1]
```diff
@@ -40,9 +40,6 @@
             android:layout_width="match_parent"
```
Comment (chrisbanes) [inline app/src/main/res/layout/activity_home.xml:40]: It's better to keep the explicit constraints and set the `layout_width` to `match_constraint` or `0dp` (same thing)

File: ui-showdetails/src/main/res/layout/view_holder_details_related_item.xml [hunk 2]
```diff
@@ -60,10 +60,6 @@
             android:padding="@dimen/spacing_small"
             android:text="@{tiviShow.title}"
             android:textAppearance="?attr/textAppearanceBody2"
-            app:layout_constraintBottom_toBottomOf="parent"
```
Comment (chrisbanes) [inline ui-showdetails/src/main/res/layout/view_holder_details_related_item.xml:63]: It's better to keep the explicit constraints and set the `layout_width` and `layout_height` to `match_constraint` or `0dp` (same thing)

File: app/src/main/res/layout/activity_home.xml (hunk 1 above)
Comment (Sathawale27) [reply app/src/main/res/layout/activity_home.xml:40]: Yeah, Right! I will do that. So, do you want me to update the layouts with `constraint layout`?

Comment (chrisbanes) [reply app/src/main/res/layout/activity_home.xml:40]: Ohhhh, I forgot that I converted these away from ConstraintLayout! 🤦

Comment (Sathawale27) [reply app/src/main/res/layout/activity_home.xml:40]: So, do you want to convert those again to ConstraintLayout? Was there any particular reason to convert away from ConstraintLayout?

Comment (chrisbanes) [reply app/src/main/res/layout/activity_home.xml:40]: No particular reason, other than simplicity.

Comment (Sathawale27) [reply app/src/main/res/layout/activity_home.xml:40]: Okay.

**Comment by chrisbanes** [comment]
Closing these, since we're focused on migrating Tivi to Compose

**Pull request closed without merging**